README.md

```diff
@@ -1,42 +1,126 @@
 
-1. Clone `vtex/toolbelt` and follow the steps on the Contributing section.
-2. Clone/Create a plugin with this template.
-3. Change the template name under this project's `package.json`.
-2. Run `yarn link` on this project.
-3. Now run `vtex link @vtex/cli-plugin-template` (or the new name) on the `vtex/toolbelt` project.
-4. Run `yarn watch` on the `vtex/toolbelt`
-5. Test the command on a VTEX IO app with `vtex-test hello`
+In the following sections, you'll learn how to make better use of this template and all the necessary steps to create any Toolbelt plugin you want.
 
-For more information, read [Ocliff Docs](https://oclif.io/docs/introduction).
+## Prerequisites
 
-[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
-![npm](https://img.shields.io/npm/v/@vtex/cli-plugin-template)
+Before proceeding any further, we recommend you to:
 
-<!-- toc -->
-* [VTEX CLI Plugin Template](#vtex-cli-plugin-template)
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
-# Usage
+1. Clone the [Toolbelt repository](https://github.com/vtex/toolbelt) into your local files.
+
+2. Check if you have the Toolbelt correctly installed in your machine.
+
+>⚠️ *We highly recommend that you install the Toolbelt using `yarn`. Follow [these instructions](https://github.com/vtex/toolbelt/#getting-started-installing-the-vtex-toolbelt) for more information on why.*
+
+3. Install the [`releasy`](https://github.com/vtex/releasy) package by running `yarn global add releasy`.
+
+4. Check our [Toolbelt design guidelines](https://github.com/vtex/ux-writing/blob/new-content/docs/text-patterns/toolbelt-cli.md) for keeping up with the **best practices** when developing your plugin.
+
+5. Check [oclif](https://oclif.io/) basic concepts.
+
+>ℹ️ Toolbelt is built with [oclif](https://oclif.io/docs/introduction), a framework for building extensible CLIs in Node.
+
+### Setting up the local development environment
+
+To locally test your plugins as you code, you'll need to use the `yarn watch` command, which is responsible for creating a [symlink](https://en.wikipedia.org/wiki/Symbolic_link) named `vtex-test`.
+
+To use the `yarn watch` command, you must add `$HOME/.vtex/dev/bin` to your `PATH` environment variable. Take the following steps:
+
+>⚠️ ***Keep in mind:** these steps may vary depending on your operating system.*
+
+1. Open the `~/.bashrc` file (or the corresponding shell configuration file).
+2. At the end of this file, add the line: `export PATH=$PATH:$HOME/.vtex/dev/bin`.
+3. Save it.
+4. Run the command source `~/.bashrc` (or corresponding file).
```

Comment:
The word `source` should be in the same syntax as `~/.bashrc`.

source `~/.bashrc` => `source ~/.bashrc`